care/facility/models/patient_base.py

```diff
@@ -44,6 +44,15 @@ def reverse_choices(choices):
     (22, "HEAD ACHE"),
     (23, "BLEEDING"),
     (24, "DIZZINESS"),
+    (25, "CHILLS"),
+    (26, "GENERAL WEAKNESS"),
+    (27, "IRRITABILITY"),
+    (28, "CONFUSION"),
+    (29, "ABDOMINAL PAIN"),
+    (30, "JOINT PAIN"),
+    (31, "REDNESS OF EYES"),
+    (32, "ANOREXIA"),
+    (33, "NEW LOSS OF TASTE/SMELL"),
```

Comment:
Waiting for clarification https://github.com/coronasafe/care_fe/issues/6942#issuecomment-1875159648